lib/testutils/src/lib.rs

```diff
@@ -238,11 +245,13 @@ impl TestWorkspace {
             signer,
         )
         .unwrap();
+        let repo_path = workspace_root.join(".jj").join("repo");
```

Comment:
nit: might be better to use `workspace.workspace_root()` or just forward to `workspace.repo_path()`. `Workspace::init*()` does path canonicalization.

Reply:
Ah, thanks for catching that. Fixed